### PR TITLE
Remove pipenv from the web scraper

### PR DESCRIPTION
**Pull request by SamDepardieu**
# Description

Replace pipenv back to a normal virtualenv.
Also include a Makefile command to setup the virtualenv.

## Type of change

- [x] :fire: Breaking change

### CODE REVIEW

File: wsf-scraper-web/Dockerfile [hunk 1]
```diff
@@ -10,9 +10,7 @@ COPY ./entrypoint.sh /wsf_scraper/entrypoint.sh
 
 COPY ./scrapy.cfg /wsf_scraper/scrapy.cfg
 
-# Pipenv require these two files to install requirements in the pinned version
-COPY ./Pipfile /wsf_scraper/Pipfile
-COPY ./Pipfile.lock /wsf_scraper/Pipfile.lock
+COPY ./requirements.txt /wsf_scraper/requirements.txt
```
Comment (nsorros) [inline wsf-scraper-web/Dockerfile]: why the duplication since you have a dockerfile.base?

Comment (SamDepardieu) [reply wsf-scraper-web/Dockerfile]: Good question, I don't remember why we kept the two files 🤔 
@hblanks Should we delete this one?

File: wsf-scraper-web/Pipfile [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
-scrapy = "==1.5.1"
-"psycopg2-binary" = "==2.7.5"
-pyahocorasick = "==1.1.8"
-requests = "==2.20.0"
-"bs4" = "*"
-"boto3" = "==1.9.1"
-sentry-sdk = "==0.6.2"
-
-[dev-packages]
-pytest = "*"
```
Comment (nsorros) [inline wsf-scraper-web/Pipfile]: this is not in the requirements? so either add it in the documentation or in the requirements

File: wsf-scraper-web/Pipfile [hunk 3]
```diff
@@ -1,19 +0,0 @@
...
-
-[requires]
-python_version = "3.6"
```
Comment (nsorros) [inline wsf-scraper-web/Pipfile]: did you update the documentation to say that you need 3.6 to run?

File: wsf-scraper-web/Pipfile (hunk 2 above)
Comment (SamDepardieu) [reply wsf-scraper-web/Pipfile]: I'm not using pytest with the scraper, so I removed it from the requirements as well.

File: wsf-scraper-web/Pipfile (hunk 3 above)
Comment (SamDepardieu) [reply wsf-scraper-web/Pipfile]: The documentation is focused on running it with docker compose, so the Python version is set in the dockerfile

Comment (hblanks) [reply wsf-scraper-web/Pipfile]: Not sure we need 3.6 to run...do we?

Comment (SamDepardieu) [reply wsf-scraper-web/Pipfile]: Any python3 is fine, I think

Comment (nsorros) [reply wsf-scraper-web/Pipfile]: Just document it is python 3 somewhere, i guess

**Pull request merged**